routes/users.js

```diff
@@ -141,6 +141,43 @@ router.get('/', authenticate, usersController.getUsers)
  */
 router.get('/self', authenticate, usersController.getSelfDetails)
 
+/**
+ * @swagger
+ * /users/isUsernameAvailable/:username:
+ *   get:
+ *     summary: check user exists or not
+ *     tags:
+ *       - Users
+ *     security:
+ *       - bearerAuth: []
+ *     responses:
+ *       200:
+ *         description: User Availability
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/userAvailable'
+ *       401:
+ *         description: unAuthorized
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/unAuthorized'
+ *       404:
+ *         description: notFound
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/notFound'
+ *       500:
+ *         description: badImplementation
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/badImplementation'
+ */
+router.get('/isUsernameAvailable/:username', authenticate, usersController.getUsernameAvailabilty)
```

Comment:
API schema yet to be added to the schema file for the new route.
https://github.com/Real-Dev-Squad/website-backend/blob/develop/README.md#api-documentation

Reply:
Updated the schema file using npm run generate-api-schema. Please let me know if any further changes needs to be done